data/tmux.conf

```diff
@@ -5,20 +5,16 @@ bind -n F1 list-keys
 
 set-option -s exit-unattached off
 set-option -g base-index 1
-set-option -g set-remain-on-exit on
```

Comment:
yeah this is new in tmux2.4, does tmux config support conditionals? If so it might be a good idea to support both ways